### PR TITLE
hwaq32: properly support openlog.

### PR DESCRIPTION
**Pull request by mlyle**
Also add it to a couple of serial ports.

### CODE REVIEW

**Comment by mlyle** [comment]
jenkins, build artifacts please

File: flight/targets/aq32/board-info/pios_board.h [hunk 1]
```diff
@@ -133,6 +133,7 @@ extern uintptr_t pios_com_storm32bgc_id;
 #define PIOS_COM_FRSKY_SPORT            (pios_com_frsky_sport_id)
 #define PIOS_COM_OPENLOG                (pios_com_openlog_logging_id)
 #define PIOS_COM_STORM32BGC             (pios_com_storm32bgc_id)
+#define PIOS_COM_OPENLOG                (pios_com_openlog_logging_id)
```
Comment (jihlein) [inline flight/targets/aq32/board-info/pios_board.h:137]: Already defined in line 134

Comment (mlyle) [reply flight/targets/aq32/board-info/pios_board.h:137]: sorry, i hacked this together in a rush and I'm apparently blind.

sx-70 was saying it doesn't work on IRC

Have you seen openlog work on aq32?

**Comment by jihlein** [comment]
Yes, use openlog and openlager all the time.  It was not defined for all uarts though, only uarts 3, 4, and 6.  I did this because on the AQ32 hardware, uart 1 is defined fr telemetry and uart 2 is defined for GPS.  They could be used for any serial protocol, it was just a choice I made at the time.

**Comment by mlyle** [comment]
Hm.  sx-70 was having issues, and i thought I spotted this define being missing as the culprit.  But I guess not.

He had hacked it in himself on uart2, so maybe there was something wrong with that.

**Comment by jihlein** [comment]
I'll put it on my todo list to add openlog to uart1 and uart2 on AQ32 target.  I think there will be less confusion overall.  I know it works for sure on uart4, as that's what I'm running it on.

**Pull request closed without merging**